src/libs/ajax/DataRepo.ts

```diff
@@ -238,12 +238,8 @@ export const DataRepo = (signal?: AbortSignal): DataRepoContract => ({
           }/search?searchText=${encodeURIComponent(searchText)}`
         );
       },
-      getConceptsHierarchy: async (_concept: SnapshotBuilderConcept) => {
-        await new Promise((resolve) => setTimeout(resolve, 1000));
-        return Promise.resolve({
-          result: getHierarchyMap(_concept.id),
-        });
-      },
+      getConceptHierarchy: async (concept: SnapshotBuilderConcept) =>
```

Comment:
I think in general I prefer more specific types over less specific ones where possible. If this was changed to accept `number` then the caller could pass in a number that wasn't a concept. Also, if we end up needing other data from `SnapshotBuilderConcept` we wouldn't have to add it.

On the other hand, if we didn't have a `SnapshotBuilderConcept` on hand already to pass, it would be a bit wasteful to make the caller create it just to call this method. In that case it might make sense to accept `number` instead.